.github/workflows/shaptools-ci.yml

```diff
@@ -138,36 +144,55 @@ jobs:
         run: make test-mlc
 
   python:
-    runs-on: ubuntu-20.04
+    runs-on: ubuntu-22.04
     strategy:
+      # do not fail if other test fails
+      fail-fast: false
       matrix:
-        os: [ubuntu-20.04]
-        python-version: [2.7, 3.6, 3.7, 3.8, 3.9]
+        container:
+          - registry.suse.com/suse/sles12sp5:latest  # python 2.7
+          - registry.suse.com/bci/bci-base:15.5      # python 3.6
+    container:
+      image: ${{ matrix.container }}
     steps:
+      - name: Auth to SCC and minimal dependencies
```

Comment:
do we need all this stuff?
I mean, I understand that testing in SLEs is good, but we already do this in OBS (where we run this exact tests as far as I know).
In my opinion it is an overkill, and a hussle, as wee need to maintain secrets now.

Reply:
As discussed, I will keep it this way for now.
It is the same CI we currently use for https://github.com/SUSE/salt-shaptools.
Trimming it down can be done another time.